node/node.go

```diff
@@ -984,6 +984,11 @@ func (n *Node) GetPeerInfo(pid string) ([]core.QueryP2PPeerInfo, error) {
 	return peerInfoSlice, nil
 }
 
+// GetConnectedPeersRatings returns the connected peers' ratings
```

Comment:
```suggestion
// GetConnectedPeersRatings returns the connected peers ratings
```
i think it's fine also without `'`, it can be updated in other referencies as well

Reply:
updated